apps/E2E/src/Text/pages/TextPageObject.ts

```diff
@@ -22,10 +66,22 @@ class TextPageObject extends BasePage {
     return By(FIRST_TEXT_COMPONENT);
   }
 
-  get _secondaryComponent() {
+  get _firstComponent() {
```

Comment:
I think we should get more specific with these names, since they are two different underlying components. The reader should know from reading this file that the `_firstComponent` has a different underlying component than `_thirdComponent`.

This helps in long-term sustainability. We have to assume that these tests are going to fail eventually, and an engineer who knows nothing about who wrote these tests will be tasked with debugging. We want to make the experience as easy as possible for that engineer (this really is lacking on our native side, which is why I think we should be extra diligent about improving it in FURN).

So perhaps these get renamed to: `_v1FirstComponent`, `_v1SecondComponent`, `_deprecatedFirstComponent`,  `_deprecatedSecondComponent`.

And perhaps add a comment about the deprecated components briefly explaining that it's in the process of deprecated, so we still need to write tests to support it